learning_resources/fixtures/schools.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "model": "learning_resources.LearningResourceSchool",
+    "fields": {
+      "id": 1,
+      "name": "School of Architecture and Planning",
+      "url": "https://sap.mit.edu/",
+      "created_on": "2024-04-17T00:00:00.000000+00:00",
+      "updated_on": "2024-04-17T00:00:00.000000+00:00"
+    }
+  },
+  {
+    "model": "learning_resources.LearningResourceSchool",
+    "fields": {
+      "id": 2,
+      "name": "School of Engineering",
+      "url": "https://engineering.mit.edu/",
+      "created_on": "2024-04-17T00:00:00.000000+00:00",
+      "updated_on": "2024-04-17T00:00:00.000000+00:00"
+    }
+  },
+  {
+    "model": "learning_resources.LearningResourceSchool",
+    "fields": {
+      "id": 3,
+      "name": "School of Humanities, Arts, and Social Sciences",
+      "url": "https://shass.mit.edu/",
+      "created_on": "2024-04-17T00:00:00.000000+00:00",
+      "updated_on": "2024-04-17T00:00:00.000000+00:00"
+    }
+  },
+  {
+    "model": "learning_resources.LearningResourceSchool",
+    "fields": {
+      "id": 4,
+      "name": "Sloan School of Management",
+      "url": "http://mitsloan.mit.edu/",
+      "created_on": "2024-04-17T00:00:00.000000+00:00",
+      "updated_on": "2024-04-17T00:00:00.000000+00:00"
+    }
+  },
+  {
+    "model": "learning_resources.LearningResourceSchool",
+    "fields": {
+      "id": 5,
+      "name": "School of Science",
+      "url": "https://science.mit.edu/",
+      "created_on": "2024-04-17T00:00:00.000000+00:00",
+      "updated_on": "2024-04-17T00:00:00.000000+00:00"
+    }
+  },
+  {
+    "model": "learning_resources.LearningResourceSchool",
+    "fields": {
+      "id": 6,
+      "name": "MIT Schwarzman College of Computing",
+      "url": "https://science.mit.edu/",
```

Comment:
I think this should be https://computing.mit.edu/ *But I'm not really sure.*

Reply:
oops, must have been a copy-paste error